graylog-configurer

```diff
@@ -89,10 +92,15 @@ if [ "${ELASTICSEARCH_CUSTOM_TEMPLATE}" ]; then
   done
 
   custom_template=$(echo "${ELASTICSEARCH_CUSTOM_TEMPLATE}" | jq -c)
-  curl -s -X PUT -H "Content-Type: application/json" -d "${custom_template}" "http://${ELASTICSEARCH_AUTHORITY}/_template/graylog-custom-template"
+  curl -sf -X PUT -H "Content-Type: application/json" -d "${custom_template}" "http://${ELASTICSEARCH_AUTHORITY}/_template/graylog-custom-template" || errors="${errors}failed to update elasticsearch custom template, "
   sleep 5
   # Rotate the active index to activate the new template
-  curl -s -X POST "${graylog_api}/system/deflector/cycle"
+  curl -sf -X POST "${graylog_api}/system/deflector/cycle" || errors="${errors}failed to cycle active index, "
+fi
+
+if [ "${errors}" ]; then
```

Comment:
```
if [[ !  -z  $errors  ]]; then
```
but I guess again just personal preference when reading a bash script

Reply:
Same as before, not a bash script. Next iteration will be in go though